.eslintrc.js

```diff
@@ -151,6 +151,16 @@ module.exports = {
       },
     },
 
+    /**
+     * New Platform client-side
+     */
+    {
+      files: ['{src,x-pack}/plugins/*/public/**/*.{js,ts,tsx}'],
```

Comment:
optional: ts linter already has `@typescript-eslint/no-var-requires` rule

Reply:
Since we want this to apply to both TS and JS, should we remove that Typescript rule?